app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -216,16 +216,10 @@ class HomeFragment : Fragment() {
             homeViewModel
```

Comment:
nit: In the extended commit summary, I would have included information on why it's okay to remove this code (though I guess it's fine if it's just in the PR text? maybe you can link to what's in the PR).